### PR TITLE
ci: auto-publish to bcr on release

### PR DESCRIPTION
**Pull request by kormide**

### CODE REVIEW

File: .github/workflows/bcr/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+bcr_test_module:
+    module_path: 'e2e/bzlmod'
+    matrix:
+        platform: ['debian10', 'macos', 'ubuntu2004', 'windows']
+    tasks:
+        run_tests:
+            name: 'Run test module'
+            platform: ${{ platform }}
+            test_targets:
+                - '//...'
```
Comment (kormide) [inline .github/workflows/bcr/presubmit.yml:10]: I couldn't find a spec for `presubmit.yml`. If we don't include build and test targets, does it automatically use `//...`?

File: .github/workflows/bcr/source.template.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "integrity": "**leave this alone**",
```
Comment (alexeagle) [inline .github/workflows/bcr/source.template.json:2]: do we actually have to have exactly this content in the value?

File: .github/workflows/bcr/presubmit.yml (hunk 1 above)
Comment (alexeagle) [reply .github/workflows/bcr/presubmit.yml:10]: take a look in Yun's https://github.com/meteorcloudy/bzlmod-examples

File: .github/workflows/bcr/source.template.json (hunk 2 above)
Comment (kormide) [reply .github/workflows/bcr/source.template.json:2]: Nope

File: .github/workflows/bcr/presubmit.yml (hunk 1 above)
Comment (kormide) [reply .github/workflows/bcr/presubmit.yml:10]: I took a look but he has the same example everywhere: https://github.com/meteorcloudy/bzlmod-examples/blob/main/registry/another/modules/A/1.1/presubmit.yml

**Review by alexeagle — APPROVED** [review]
Is it good to have so much duplication between the two bcr*.yml files vs. having both workflows defined in one?

**Comment by kormide** [comment]
> Is it good to have so much duplication between the two bcr*.yml files vs. having both workflows defined in one?

It's not great. The publishing to our own bcr was sort of an after thought, and I didn't want to mix them in case we bring the real bcr one into bazel-contrib.

**Comment by kormide** [comment]
> > Is it good to have so much duplication between the two bcr*.yml files vs. having both workflows defined in one?
> 
> It's not great. The publishing to our own bcr was sort of an after thought, and I didn't want to mix them in case we bring the real bcr one into bazel-contrib.

Oh and there would be a lot of repetition anyway since the create pr plugin commits the working directory and publishes the pr all in one. We'd have to redo most of the steps to form the bcr entry in the working directory to call it again.

**Pull request merged**